src/containers/GradebookPage/index.jsx

```diff
@@ -63,12 +64,16 @@ export class GradebookPage extends React.Component {
 }
 GradebookPage.defaultProps = {
   location: { search: '' },
+  showBulkManagement: false,
 };
 GradebookPage.propTypes = {
+  history: PropTypes.shape({
+    push: PropTypes.func,
+  }).isRequired,
   location: PropTypes.shape({ search: PropTypes.string }),
   courseId: PropTypes.string.isRequired,
   initializeApp: PropTypes.func.isRequired,
-  showBulkManagement: PropTypes.bool.isRequired,
+  showBulkManagement: PropTypes.bool,
```

Comment:
why?

Reply:
can return undefined, which causes error message but not failure.